plugins/out_datadog/datadog.c

```diff
@@ -157,10 +189,18 @@ static int datadog_format(const void *data, size_t bytes,
         }
 
         /* Append initial object k/v */
-        int i = 0;
+        int i = 0, ind;
```

Comment:
looking at the code style convention you should set `ind = 0;`

Reply:
changed